app/Resources/assets/scss/main.scss

```diff
@@ -144,6 +144,24 @@ body#homepage {
 // Page: 'login'
 // --------------------------------------------------
 body#login {
+    #login-users-help p {
```

Comment:
well, then the id ``login-users-*`` would be weird (a 403 page would not display a login form btw)